src/Time/Units.hs

```diff
@@ -144,18 +142,18 @@ __Usage example:__
 "3h5m3s"
 
 -}
-type family (...) (a :: Rat) (b :: Rat) where
+type family (from :: Rat) ... (to :: Rat) where
     x ... y = TakeWhileNot y (DropWhileNot x AllTimes)
 
 -- Drops wile not the required time unit in 'AllTimes'.
-type family DropWhileNot (a :: Rat) (c :: [Rat]) where
+type family DropWhileNot (from :: Rat) (units :: [Rat]) where
```

Comment:
Could you please write result kinds of thess type families as well? I personally find it more useful. There's some type-level magic is going on here so I would like to have kind signatures for both arguments and result be explicit... With kinds specified explicitly it's easier to understand what's going on (heyeayayeayyey)